### PR TITLE
Devops/single recipe page (PRA-112)

### PR DESCRIPTION
**Pull request by AnnaPestova1**
## One Line Description
Render page for single saved recipe
## Requirements
save new recipe (to reduce any errors with previously added recipes due backend changes)
_What this code should do_
on the page with saved recipes click on the recipe you would like to see, the new window should be open
## Notes
the logic will be added in next pull request
_Any additional info_

## Test Steps

_Instructions on how to test the changes_

## UI/UX Outcome

_Attach a screenshot here_
image

## Checklist

- The Linear ID is in the PR title
- My code follows best practices
- Documentation is included where needed

### CODE REVIEW

**Review by AnnaSolovykh — APPROVED** [review]
Looks great!

File: src/components/SingleRecipePage.tsx [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+        </GridItem>
+        <GridItem colSpan={1} w="100%" position="relative">
+          <Flex
+            w="100%"
+            gap="2"
+            flexShrink="1"
+            flexWrap="wrap"
+            position="absolute"
+            bottom="2"
+            // alignSelf=""
```
Comment (rebekahcallkacz) [inline src/components/SingleRecipePage.tsx:92]: NON-BLOCKING: Do you still need this?

File: src/components/SingleRecipePage.tsx [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+                  {
+                    displayName: "Protein",
+                    content: recipe.recipeNutritionInfo.NutritionInfoProtein,
+                    unit: "g"
+                  },
+                  {
+                    displayName: "Fat",
+                    content: recipe.recipeNutritionInfo.NutritionInfoFat,
+                    unit: "g"
+                  }
```
Comment (rebekahcallkacz) [inline src/components/SingleRecipePage.tsx:180]: NON-BLOCKING: I like this format, but for easier reading, you can store this array in a constant at the top of this component to make the `return` statement easier to read.

**Review by rebekahcallkacz — APPROVED** [review]
Tested locally and looks good to me!

File: src/components/SingleRecipePage.tsx [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+      <Grid templateColumns="repeat(3, 1fr)" gap={6}>
+        <GridItem colSpan={2} w="100%">
+          <Flex marginTop="10" marginBottom="5" alignItems="center" gap={2}>
+            <IconButton
+              size="lg"
+              variant="ghost"
+              aria-label="on the previous page"
+              icon={<ArrowBackIcon />}
+              title="on the previous page"
+            />
```
[the remark this answers is not in the document]
Comment (AnnaPestova1) [reply src/components/SingleRecipePage.tsx:55]: Hi! Thanks for review! I am adding button functionality in the next branch I am working right now)

File: src/components/SingleRecipePage.tsx (hunk 1 above)
Comment (AnnaPestova1) [reply src/components/SingleRecipePage.tsx:92]: Thanks! I missed it! I'll remove in the branch I am working right now.

File: src/components/SingleRecipePage.tsx (hunk 2 above)
Comment (AnnaPestova1) [reply src/components/SingleRecipePage.tsx:180]: Thank you! I will do it  in the branch I am working right now.

**Pull request merged**